train.py

```diff
@@ -188,7 +188,7 @@ def compile_model_mlp(geneparam, nb_classes, input_shape):
 
     return model
 
-def compile_model_cnn(geneparam, nb_classes, input_shape):
+def compile_model_cnn(genome, nb_classes, input_shape):
```

Comment:
`nb_neurons()` is now a property of `genome`. We have to pass the entire object if we want to access/print the array of neurons at each layer nicely.